app/views/admin/legacy/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Admin: Menu</h1>
```

Comment:
Not sure why the git diff picked this up as a deleted & completely new file, instead of it being moved.